src/Symfony/Component/Lock/Store/DoctrineDbalPostgreSqlStore.php

```diff
@@ -268,7 +268,7 @@ private function filterDsn(#[\SensitiveParameter] string $dsn): string
         }
 
         [$scheme, $rest] = explode(':', $dsn, 2);
-        $driver = strtok($scheme, '+');
+        $driver = substr($scheme, 0, strpos($scheme, '+') ?: \strlen($scheme));
```

Comment:
now we can :)

```suggestion
        $driver = substr($scheme, 0, strpos($scheme, '+') ?: null);
```

Reply:
I hesitated to ensure the person who potentially back ports doesn't forget about it. But tests will fail if it's not replaced by `strlen()`, so here you go!